bin/gather_logs.sh

```diff
@@ -47,11 +48,10 @@ sort_by_last() {
 	-o 'UserKnownHostsFile /dev/null')
     logs=(/var/log /etc)
     logs+=(/var/chef/cache /var/cache/chef /opt/dell/crowbar_framework/db)
-    curlargs=(-o /dev/null -D - --connect-timeout 30 --max-time 120 --insecure \
-    --location)
-    [[ $CROWBAR_KEY ]] && curlargs+=(--digest -u "$CROWBAR_KEY")
-    for to_get in nodes proposals roles; do
-	curl "${curlargs[@]}" "http://localhost/$to_get" || :
+    crowbarctl node list -U machine-install -P $CROWBAR_PASS
+
+    for to_get in proposals roles; do
+        crowbarctl $to_get proposal list crowbar
```

Comment:
Why do we need `-U machine-install -P $CROWBAR_PASS` a few lines earlier and not here? Isn't it simply missing?

Reply:
Ah, I guess I missed this